API/Program.cs

```diff
@@ -259,8 +259,8 @@ await context.HttpContext.Response.WriteAsync(
             Version = "v1",
             Title = "osu! Tournament Rating API",
             Description = "The official resource for reading and writing data within the osu! Tournament Rating platform.",
-            TermsOfService = new Uri(
-                "https://github.com/osu-tournament-rating/otr-wiki/blob/master/api/usage/limits/en.md"
+            TermsOfService = new Uri( // Stage will change this later when there is actually a link for TOS
+                "https://github.com/osu-tournament-rating"
```

Comment:
This should either be removed completely for now, or replaced with a link straight to the API docs.

Reply:
i removed comment and linked straight to docs